src/js/utils/browser.js

```diff
@@ -70,6 +70,11 @@ export function isMobile() {
     return isIOS() || isAndroid();
 }
 
+export function isTouchScreen() {
```

Comment:
Is this a check for touch-only devices? Or devices with multi pointer support (mouse, touch, pointer)? (`hasTouch` vs `isTouch` vs `feature.touch` `feature.touch`)

Why `isMobile`? What do we want for Windows?

Remove the word 'Screen'.